### ISSUE TITLE
[Xiaomi Mi11X Pro (haydn)] Bootloop after installing IMS

### ISSUE THREAD

**Issue by ravikiranp123** [phhusson/treble_experimentations#1974]
Hello! I tried AOSP and pretty much evertything works on first boot. However, if I install IMS from phh settings, and reboot. it bootloops. Without IMS fix, calling does not work. I would be grateful if anyone could take a look into the logs.

I have included 3 logs. 
1. Before installing IMS
[1. pre_ims_log.txt](https://github.com/phhusson/treble_experimentations/files/7020063/1.pre_ims_log.txt)
[1a. pre_ims_log2.txt](https://github.com/phhusson/treble_experimentations/files/7020064/1a.pre_ims_log2.txt)

2. During installation
[2. installing_ims_log.txt](https://github.com/phhusson/treble_experimentations/files/7020065/2.installing_ims_log.txt)

3. Post installation when it bootloops.
[3. post_ims_install_bootloop_log.txt](https://github.com/phhusson/treble_experimentations/files/7020066/3.post_ims_install_bootloop_log.txt)

Any help is greatly appreciated.

**Comment by phhusson** [comment]
The crash isn't related to IMS.
Please reboot few times without IMS, and see if that boots fine

**Comment by ravikiranp123** [comment]
> The crash isn't related to IMS.
> Please reboot few times without IMS, and see if that boots fine

Yup. That works just fine. There are only two changes I have done. One is to disable audio effects and then reboot multiple times. Works just fine. Then I install IMS and reboot. The first reboot tends to work fine and calling works as well. Subsequent reboots sit at the Xiaomi logo. The android bootlogo doesn't even start.

Also, what do you believe is the issue here?
